packages/material-ui-styles/src/install.js

```diff
@@ -1,4 +1,3 @@
-/* istanbul ignore file */
```

Comment:
I'm removing all the istanbul ignore, @eps1lon is right, let's leave the 100% utopia.

Reply:
To be fair that broken window analogy you linked made total sense to me. I just think that these comments won't help in that regard. At least this way we have the actual number to look at. Previously it was 100% (kind of; who knows how good it is actually). Now we know how good/bad we are and have tools helping us analyze it. Codesearch is not really that helpful IMO. It's like prefering `// TODO` over creating issues.

Reply:
I agree.